web-components/src/components/cards/ProjectCard/ProjectCard.tsx

```diff
@@ -118,7 +120,7 @@ export function ProjectCard({
     startIcon: buttonStartIcon,
     className: buttonClassName,
   } = cardButton;
-  const isButtonDisabled =
+  const isCreatePostButtonDisabled =
```

Comment:
I don't think we should rename this var because it's not necessarily always a "create post" button